clang/include/clang/Basic/Attr.td

```diff
@@ -1313,6 +1313,40 @@ def SYCLIntelNoGlobalWorkOffset : InheritableAttr {
   let PragmaAttributeSupport = 0;
 }
 
+def SYCLIntelLoopFuse : InheritableAttr {
+  let Spellings = [CXX11<"intel","loop_fuse">];
+  let Args = [ExprArgument<"Value", /*optional=*/ 1>];
```

Comment:
Based on some other parts of the patch, I sort of wonder whether it makes more sense to use a `DefaultIntArgument<"Value", 1>` argument instead? The syntactic attribute the user would write would still be optional as it is, but the semantic attribute used by the rest of the frontend wouldn't have to worry about "is the argument specified, if not, do this instead" and could just assume there's always an integer argument provided.

Reply:
Oh, good call! I think you do need to stick with `ExprArgument` in that case.